pom.xml

```diff
@@ -34,8 +34,8 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
```

Comment:
can you also update the github pipeline - we no longer need to compile with java 8 in this case

Reply:
Updated the Pipeline. Builds Uploads and Analyzes only with Java 11 now. Code coverage also works with Java 11 now